packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -59,28 +88,30 @@ function getPreviousChar(doc: Document, end: number) {
 function getNextChar(doc: Document, start: number) {
   let end = start;
 
-  while (end < doc.content.length) {
-    let parseAnnotations = doc.where(
-      a => a instanceof ParseAnnotation && a.start <= end && a.end > end
+  function isOverlappingParseAnnotation(a: Annotation<any>) {
```

Comment:
These functions are being allocated every time this function gets called; could we move these to the outer scope?

Reply:
This function closes over a local variable so that would require a more significant refactor